### ISSUE TITLE
Support for passage

### ISSUE THREAD

**Issue by peterhoeg** [IJHack/QtPass#597]
**Is your feature request related to a problem? Please describe.**
passage is a fork of pass that supports using age rather than GPG as the backend. 

The passage binary supports the same arguments as pass, so it would be great if QtPass could do the following:

Defer all operations to the binary (example: pass ls and passage ls to show all items instead of looking for .gpg on its down) and have a configurable file extension (.age vs .gpg).

**Describe the solution you'd like**
Basically, regardless of the choice of backend (pass and passage), have QtPass work the same.

**Comment by peterhoeg** [comment]
Related: https://bugs.kde.org/show_bug.cgi?id=459051
